cluster/juju/layers/kubernetes-common/lib/charms/layer/kubernetes_common.py

```diff
@@ -383,6 +383,12 @@ def _cloud_endpoint_ca_path(component):
     return _snap_common_path(component) / 'cloud-endpoint-ca.crt'
 
 
+def encryption_config_path():
+    apiserver_snap_common_path = _snap_common_path('kube-apiserver')
+    encryption_conf_dir = apiserver_snap_common_path / 'encryption'
+    return encryption_conf_dir / 'encryption_config.yaml'
```

Comment:
It looks like this function is only used by kubernetes-master. Can it go there?

Reply:
Yeah, it could, but then I'd have to export `_snap_common_path`.  I figured I'd keep it with all of the other path helpers and avoid having to change the export even though it's not really "common."

Reply:
Works for me. :+1: